src/main.rs

```diff
@@ -556,37 +548,41 @@ fn load_console_sockets() -> Result<(RawFd, RawFd)> {
             Ok(()) => csocketfd,
         };
     let console = "console";
-    let consolefd = match open(&*console, O_NOCTTY | O_RDWR, Mode::empty()) {
-        Err(e) => {
-            if e != Sys(Errno::ENOENT) {
-                let msg = format!("failed to open {}", console);
-                return Err(e).chain_err(|| msg)?;
+    let consolefd =
+        match open(&*console, OFlag::O_NOCTTY | OFlag::O_RDWR, Mode::empty()) {
+            Err(e) => {
+                if e != ::nix::Error::Sys(Errno::ENOENT) {
+                    let msg = format!("failed to open {}", console);
+                    return Err(e).chain_err(|| msg)?;
+                }
+                -1
             }
-            -1
-        }
-        Ok(fd) => fd,
-    };
+            Ok(fd) => fd,
+        };
     Ok((csocketfd, consolefd))
 }
 
 fn finish_create(id: &str, dir: &str, matches: &ArgMatches) -> Result<()> {
-    let spec = Spec::load(CONFIG).chain_err(
-        || format!("failed to load {}", CONFIG),
-    )?;
+    let spec =
+        Spec::load(CONFIG).chain_err(|| format!("failed to load {}", CONFIG))?;
 
     let rootfs = canonicalize(&spec.root.path)
         .chain_err(|| format!{"failed to find root path {}", &spec.root.path})?
         .to_string_lossy()
         .into_owned();
 
```

Comment:
this section looks like it is pulling in old code that was removed

Reply:
The note was removed. I'm not sure if changing the name csock to csocket has any effect but it seems like an unnecessary change.

Reply:
Felt unnatural to have `csock` but also `tsocket`.